internal/options/options.go

```diff
@@ -55,10 +57,12 @@ func (o *Options) AddFlags() {
 		o.flags.PrintDefaults()
 	}
 
-	o.flags.StringVar(&o.Host, "host", "0.0.0.0", `Host to expose Events metrics on.`)
-	o.flags.IntVar(&o.Port, "port", 8080, `Port to expose Events metrics on.`)
-	o.flags.StringVar(&o.ExporterHost, "exporter-host", "0.0.0.0", `Host to expose kube-events-exporter own metrics on.`)
-	o.flags.IntVar(&o.ExporterPort, "exporter-port", 8081, `Port to expose kube-events-exporter own metrics on.`)
+	o.flags.StringVar(&o.Apiserver, "apiserver", "", "The URL of the apiserver to use as a master.")
+	o.flags.StringVar(&o.Kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file.")
```

Comment:
Could it also read kubeconfig from KUBECONFIG env var? Seems like this is fairly common practice